dot/telemetry/telemetry.go

```diff
@@ -65,13 +66,16 @@ func GetInstance() *Handler {
 				}
 				log.SetOutput(&handlerInstance.buf)
 				log.SetFormatter(new(MyJSONFormatter))
+				go handlerInstance.sender()
 			})
 	}
 	return handlerInstance
 }
 
 // AddConnections adds connections to telemetry sever
 func (h *Handler) AddConnections(conns []*genesis.TelemetryEndpoint) {
+	h.Lock()
```

Comment:
This lock may not be required as we are adding connections only when starting the node.

Reply:
Removed lock.